app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -32,6 +32,11 @@ data class RecentPatient(
   @Dao
   interface RoomDao {
 
+    /**
```

Comment:
Can you mention what "RecentPatient" means first and then describe the query?

Reply:
Also, it is alright if the query is not described at all. That is more of a good to have.

Reply:
I've updated the comment.